src/views/workflow-page/workflow-page-status-tag/workflow-page-status-tag.types.ts

```diff
@@ -0,0 +1,13 @@
+import type { PageTab } from '@/components/page-tabs/page-tabs.types';
```

Comment:
These types seem tab related, shouldn't they be in the workflow-page-tabs folder?